dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/parameters/Parameters.java

```diff
@@ -22,6 +22,7 @@
 import org.springframework.data.mongodb.core.mapping.Field;
 
 import java.util.List;
+import java.util.Map;
```

Comment:
is this import needed? Don't see uses of Map in the introduced changes in this class